src/main/java/com/hillel/items_exchange/model/Message.java

```diff
@@ -0,0 +1,39 @@
+package com.hillel.items_exchange.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import javax.persistence.*;
+import java.time.LocalDateTime;
+import java.util.List;
+
+@Entity
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+public class Message extends BaseEntity {
+    @ManyToOne
+    @JoinColumn(name = "chat_id", nullable = false)
+    private Chat chat;
+    @ManyToOne
+    @JoinColumn(name = "user_id")
+    private User user;
+    private String text;
+    @OneToMany(mappedBy = "message", cascade = CascadeType.ALL, orphanRemoval = true)
+    List<Attachment> attachments;
+
+    @PrePersist
+    private void onPrePersist() {
+        setCreated(LocalDateTime.now());
+        setUpdated(LocalDateTime.now());
```

Comment:
For setting time during create or update operations we have CustomDateTimeProvider. You could update it for LocalDateTime and tried if it works.

Reply:
Ok. Will try this.